docker-app/qfieldcloud/core/models.py

```diff
@@ -1478,7 +1478,12 @@ def clean(self) -> None:
             if self.collaborator.is_person:
                 members_qs = organization.members.filter(member=self.collaborator)
 
-                if not members_qs.exists():
+                # for organizations-owned projects, the candidate collaborator
+                # must be a member of the organization or the organization's owner
```

Comment:
```suggestion
                # for organizations-owned projects, the candidate collaborator
                # must be a member of the organization or and not organization's owner
```

Reply:
@why-not-try-calmer isn't it more correct now?

Reply:
As of this PR, the `if` above will **not** lead to a ValidationError if the candidate collaborator is the organization's owner.

Reply:
Ok, I see, I misread the `not` in front of the condition.